sandbox/registration-processor-mz.properties

```diff
@@ -539,7 +539,7 @@ mosip.regproc.packet.classifier.eventbus.kafka.poll.frequency=100
 mosip.regproc.packet.classifier.eventbus.kafka.group.id=packet-classifier-stage
 mosip.regproc.packet.classifier.message.expiry-time-limit=${mosip.regproc.common.stage.message.expiry-time-limit}
 
-#quality-checker-stage
+#quality-classifier-stage
 mosip.regproc.quality.checker.eventbus.kafka.commit.type=single
```

Comment:
You have replaces all places of mosip.regproc.quality.checker with mosip.regproc.quality.classifier in the properties

Reply:
Change kafka group id also

Reply:
changed other other checker props, couldnt change server.port, eventbus.port and servlet.path, as didnt seen where exactly these props were used.

And need to know where in code the kafka id value will impact, as stage name has been modified.

Reply:
it is used from registration-processor-core project, also the property prefix is appended and used, so direct text search will not help to find here